pkg/cmd/pulumi/main_test.go

```diff
@@ -20,14 +20,25 @@ import (
 	"io"
 	"os"
 	"path"
+	"reflect"
 	"testing"
+	"time"
 
 	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
 	"github.com/spf13/pflag"
 )
 
 type noTestDeps int
 
+type corpusEntry = struct {
+	Parent     string
+	Path       string
+	Data       []byte
+	Values     []interface{}
+	Generation int
+	IsSeed     bool
+}
```

Comment:
unexported type from testing package, we need an alias definition to implement the methods below.